multimedia/fadecandyserver/Makefile

```diff
@@ -0,0 +1,54 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=fadecandyserver
+PKG_VERSION:=2.1.05
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://codeload.github.com/bluewavenet/fadecandyserver/tar.gz/v$(PKG_VERSION)?
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+#PKG_HASH:= #shasum -a 256 of tar.gz of source files goes here
+PKG_HASH:= 4033711b2b21ade83013828b383ac386716b9f724e6607cb2ef4cb4b76787c71
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+
+PKG_MAINTAINER:=Rob White <rob@blue-wave.net>
+PKG_BUILD_PARALLEL:=1
+PKG_LICENSE:=MIT
+
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/fadecandyserver
+	SECTION:=multimedia
```

Comment:
Replace tab with 2 spaces.

Reply:
Consistency between packages.

Reply:
ok
Any rule on where to use 2 spaces and where tabs are acceptable?